system/CLI/Commands.php

```diff
@@ -52,12 +52,12 @@ public function __construct($logger = null)
     /**
      * Runs a command given
      *
-     * @return int|void Exit code
+     * @return int|null Exit code
      */
     public function run(string $command, array $params)
     {
         if (! $this->verifyCommand($command, $this->commands)) {
-            return;
+            return null;
```

Comment:
I think this should return an exit code too. `EXIT_ERROR`?

Reply:
updated to return EXIT_ERROR https://github.com/codeigniter4/CodeIgniter4/pull/9311/commits/dcf235ef3e6663be69573b497013f89afafd1463